### DISCUSSION TITLE
Welcome to CGNStalk!

### DISCUSSION THREAD

**Discussion by brtnfld** [CGNS/CGNS#248] [CGNStalk]
## 👋 Welcome!
  CGNStalk _Discussions_ has been set up to facilitate the exchange of information among CGNS users.

**Comment by poinot** [comment]
**Great!**

Thanx @brtnfld &  @MicK7  for the setup!

Nice new @CGNS mailing list, thanks to [NASA](https://mars.nasa.gov/technology/helicopter/) for hosting [ye old good mailman](https://en.wikipedia.org/wiki/GNU_Mailman) for years!

- [x]  I'm going to ask
- [x]  I'm going to answer
- [ ]  I'm going to send ads & other spam stuff

**Reply by MicK7** [reply]
All the thanks and praises should go to Scot

**Comment by crumsey** [comment]
We have a lot of “watchers” on NASA’s CGNSTalk… people who are interested in CGNS-related discussions, but they are by no means “contributers”.

What do these types of people have to do to be a part of the new discussions?  Do they have to have a github.com account, and subscribe to CGNS?  Do they have to be set up as part of the “contributer” team?  Should we have a “watcher” team set up for this instead?  Or do they not have to be a member of any CGNS team?  It’s not clear to me what actions a person needs to take in order to receive discussions by email (if desired), if they are currently not a member of github.com.

Chris

**Reply by brtnfld** [reply]
As far as I know, you would have to have a GitHub account and watch the CGNS/CGNS repo (with the discussion option set) to get a notification (by email if that is how you have your GitHub account settings).

They don't need to be a member of any team.

**Reply by poinot** [reply]
Probably these people are not interested with **all** emails.

We could have a separate 'hard-coded' list and send a digest with release notes, or maybe have a **news** page everybody could read (and add an RSS on this page?)

**Reply by MicK7** [reply]
With current setup you can use the "custom watch" feature to monitor either Release or Discussion or both.
So this is quite close to the current CGNS mailing list. Then to not be spammed too much one can unsubscribe to topics. To keep a discussion less noisy the maintainer could also make the discussion private but it would defeat the goal of the forum as an open place.

**Reply by brtnfld** [reply]
I did not see the suggestion in discussions feedback for a digest mode,

https://github.com/github/feedback/discussions/categories/discussions-feedback

so I think we would have to suggest that feature there.
